vertx-grpc-docs/src/main/java/examples/grpc/StreamingGrpcService.java

```diff
@@ -101,86 +101,43 @@ public static Service of(StreamingService service) {
     all.add(Source);
     all.add(Sink);
     all.add(Pipe);
+    all.add(SourceTranscodingMethod);
```

Comment:
the generator should only create a single method whatsoever, that is `Source` we don't need repetition, we don't need to create two per service